deploy/src/main/deploy/helm/templates/hono-adapter-amqp/hono-adapter-amqp-vertx-deployment.yaml

```diff
@@ -20,6 +20,12 @@ spec:
         version: "{{ .Chart.AppVersion }}"
         group: {{ .Values.project.groupId }}
     spec:
+      {{- if eq .Values.deployJaeger true}}
+      initContainers:
+        - name: wait-for-jaeger-agent
+          image: busybox:1.28
+          command: ['sh', '-c', 'until nslookup {{ .Values.jaeger.agentHost }}; do echo waiting for {{ .Values.jaeger.agentHost }}; sleep 2; done;']
+      {{- end}}
```

Comment:
Note: this has been added to make sure the Jaeger client can resolve the `jaeger-agent.hono` hostname when the tracer is initialized. 
Otherwise the tracer instance would be initialized in such a way that the tracer effectively does nothing (`Using NoopSender, meaning that data will not be sent anywhere!`).